### PR TITLE
Patch 3

### PR DESCRIPTION
**Pull request by jbondguy007**
Two new files:
- _emoticonresponse.siml: Simple responses to simple text-only emoticons
- _jbond.siml: Misc personal file

### CODE REVIEW

File: Brain/ArchiBoT/_jbond.siml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Siml>
+	<Concept Name="_jbond" Type="Public">
+		
+		<Model>
+			<Pattern>RIP</Pattern>
+			<Response>:ripgocha:</Pattern>
```
Comment (Thielak) [inline Brain/ArchiBoT/_jbond.siml]: It should be `<Response>:ripgocha:</Response>`

File: Brain/ArchiBoT/_emoticonresponse.siml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+					<Item>You're all smiles, <User Get="name" />.</Item>
+				</Random>
+			</Response>
+		</Model>
+		
+		<Model>
+			<Pattern>
+				<Item>._.</Item>
+				<Item>&gt;_&gt;</Item>
+				<Item>&lt;_&lt;</Item>
```
Comment (Thielak) [inline Brain/ArchiBoT/_emoticonresponse.siml]: I don't think \< and \> will work

Comment (JustArchi) [inline Brain/ArchiBoT/_emoticonresponse.siml]: Why? Did you test it? It looks like fine XML to me.

File: Brain/ArchiBoT/_jbond.siml (hunk 1 above)
Comment (JustArchi) [inline Brain/ArchiBoT/_jbond.siml]: Good catch!

Comment (jbondguy007) [inline Brain/ArchiBoT/_jbond.siml]: Damn, and I was sure I made no mistakes this time. Thanks for the heads up.

**Pull request merged**